apps/docs/package.json

```diff
@@ -15,12 +15,16 @@
 		"test": "npm run test:unit -- --run"
 	},
 	"devDependencies": {
-		"@sveltejs/adapter-cloudflare": "^4.8.0",
-		"@sveltejs/kit": "^2.12.1",
-		"@sveltejs/vite-plugin-svelte": "^5.0.2",
+		"@sveltejs/adapter-cloudflare": "^4.9.0",
+		"@sveltejs/kit": "^2.14.0",
+		"@sveltejs/vite-plugin-svelte": "^5.0.3",
 		"@sveltepress/theme-default": "^5.0.5",
 		"@sveltepress/vite": "^1.1.2",
+		"@typescript/vfs": "^1.6.0",
 		"autoprefixer": "^10.4.20",
-		"svelte": "catalog:"
+		"runed": "^0.22.0",
+		"svelte": "catalog:",
+		"svelte-docgen": "workspace:*",
+		"svelte2tsx": "^0.7.31"
```

Comment:
nit: I couldn't find where we're using `svelte2tsx` in the `apps/docs`.

We can consider using [knip](https://github.com/webpro-nl/knip) in the project workspace, so it can catch those issues.

Reply:
Actually, it’s a bit tricky to spot, but we’re using it here to load `svelte-shims-v4.d.ts`:

https://github.com/svelte-docgen/svelte-docgen/pull/45/files#diff-e04f75dbb1fcf9fb697431b29493888d6950b9a6994b5b929dce2d19766c1586R6

(knip looks like a great tool.)